nttt/tag_trimming.py

```diff
@@ -1,11 +1,21 @@
 import re
+from .nttt_logging import nttt_display_tags
 
 
-def trim_tags(file_content):
-    return re.sub(r'<(?P<tag>[\w\d]+?)>(?P<content>.+?)</(?P=tag)>', replacement_builder, file_content)
+def trim_tags(file_content, logging="off"):
+    if logging == "off":
+        return re.sub(r'<(?P<tag>[\w\d]+?)>(?P<content>.+?)</(?P=tag)>', replacement_builder, file_content)
+    elif logging == "on":
+        return re.sub(r'<(?P<tag>[\w\d]+?)>(?P<content>.+?)</(?P=tag)>', replacement_builder_logging, file_content)
 
 
 def replacement_builder(matchobj):
     tag_name = matchobj.group("tag")
     stripped_content = matchobj.group("content").strip()
+    return "<{}>{}</{}>".format(tag_name, stripped_content, tag_name)
+
+def replacement_builder_logging(matchobj):
```

Comment:
Currying (see `cleanup_markdown.py`) would help here to avoid code duplication

Reply:
I transferred currying technique to tag_trimming module